### ISSUE TITLE
Use of `att.slashCount` and definition of `data.SLASH`

### ISSUE THREAD

**Issue by lpugin** [music-encoding/music-encoding#1283]
There is an inconsistency with `mensur@slash` and `staffDef@mensur.slash` (defined in `att.mensural.vis`, namely that the first is a member of `att.slashCount`  for `@slash` while `att.mensural.vis` defines `mensur.slash` directly as a `<rng:data type="positiveInteger"/>`.

The other question is the definition of `data.SLASH` used by `att.slashCount` that reads `The number of slashes to be rendered for tremolandi.`, which obviously does not make sense.

**Comment by pe-ro** [comment]
Hi @lpugin, this is obviously a place where a bit of experimentation didn't get cleaned up.  I suggest replacing the definition of `@mensur.slash` in `att.mensural.vis` with
```
<macroSpec ident="data.SLASH" module="MEI" type="dt">
<desc xml:lang="en">The number of slashes to be rendered.</desc>
<content>
<rng:data type="positiveInteger">
<rng:param name="minInclusive">1</rng:param>
<rng:param name="maxInclusive">6</rng:param>
</rng:data>
</content>
</macroSpec>
```
and replacing the definition of `@mensur.slash` in `att.mensural.vis` with 

```
<attDef ident="mensur.slash" usage="opt">
<desc xml:lang="en">Indicates the number lines added to the mensuration sign. For example, one slash is added for what we now call 'alla breve'.</desc>
<datatype>
<rng:ref name="data.SLASH"/>
</datatype>
</attDef>
```

**Comment by rettinghaus** [comment]
`data.SLASH` was originally used for tremolos (see a6f4335), and obviously only reused for mensural notation. As the model has been changed for tremolos it should be time now to have a better data type here. 
In the literature I didn't find more than three lines mentioned, so `maxInclusive` should be 3 in this case. Also, nobody seems to write about slashes, but strokes. So maybe `data.STROKE` or `data.MENSURSTROKE` would be more appropriate? 
Furthermore looking at the respective [SMuFL range](https://w3c.github.io/smufl/latest/tables/medieval-and-renaissance-prolations.html) I wonder if there should be the possibility to encode the direction of the strokes. 
Any comments from @annplaksin or @martha-thomae?

**Comment by pe-ro** [comment]
@rettinghaus, a search of the web shows that "stroke" and "slash" are often used interchangeably, as in this passage from the Wikipedia article on "tremolo" --
```
In musical notation, tremolo is usually notated as regular repeated
demisemiquavers using strokes through the stems of the notes. Generally, 
there are three strokes, except on notes which already have beams 
or flags: quavers then take two slashes, and semiquavers take one.
```
I haven't performed an extensive analysis, but perhaps "slash" is used more in the lay community and "stroke" is used by those who want to sound more "professional".  However, it's an overstatement to say "no one" uses the term "slash".  That being said, I don't think it matters which term is used for tremolo notation as long as it's well-defined and employed consistently.  Just in case it needs saying, I don't recommend, however, replacing the term "slash" with "stroke" in the context of rhythmic slash notation; that is,
![image](https://github.com/music-encoding/music-encoding/assets/5712877/7549f760-8135-4709-b92a-2376ff7cf196)

**Comment by rettinghaus** [comment]
Yes, @pe-ro, but we're not talking about tremolos or repeats anymore, this is only about mensural music now.

**Comment by ahankinson** [comment]
> The slash is the oblique slanting line punctuation mark /. Also known as a stroke, a solidus, a forward slash or several other historical or technical names including oblique and virgule.

https://en.wikipedia.org/wiki/Slash_(punctuation)

Some sources also say that "stroke" is British, while "slash" is American. Since we've standardised on American English in MEI, then we should probably keep with "slash".
